Frameworks/CoreText/CTRun.mm

```diff
@@ -44,18 +44,27 @@ - (id)init {
     return self;
 }
 
+- (instancetype)copyWithZone:(NSZone*)zone {
+    _CTRun* ret = [_CTRun new];
+    ret->_attributes = [_attributes copy];
```

Comment:
>_attributes = [_attributes copy]; [](start = 9, length = 33)

I don't think CoreText has ARC enabled does it? If not, this and the copy below would leak. Use .attach() instead of =, or call autorelease